src/VersionParser.php

```diff
@@ -50,7 +50,7 @@ class VersionParser
      */
     public static function parseStability($version)
     {
-        $version = preg_replace('{#.+$}i', '', $version);
+        $version = preg_replace('{#.+$}', '', $version);
```

Comment:
Isn't version is case insensitiv?

Reply:
It doesn't matter if the regex doesn't check on any alphabet character.